### ISSUE TITLE
RemoteAdministrationEndpoint attribute for aws_fsx_windows_file_system

### ISSUE THREAD

**Issue by JuanOI** [terraform-providers/terraform-provider-aws#13616]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

FSx has two network interfaces attached to it. One is used by clients by using the DNS Name attribute and the other is used for remote PowerShell connections.

Currently, only the DNS name is an attribute supported by the `aws_fsx_windows_file_system` resource. I would like to have the `RemoteAdministrationEndpoint` (Windows Remote PowerShell Endpoint) also available as an attribute.

### New or Affected Resource(s)

* aws_fsx_windows_file_system

### Potential Terraform Configuration

Addition of a `remote_admin_endpoint` attribute to the resource which can be used to retrieve the `RemoteAdministrationEndpoint`. This could then be referenced as

```HCL

output "my_fsx_admin_endpoint" {
  value = aws_fsx_windows_file_system.my_fsx.remote_admin_endpoint
}

```

### References

* https://docs.aws.amazon.com/fsx/latest/WindowsGuide/remote-pwrshell.html#using-rps
* https://docs.aws.amazon.com/cli/latest/reference/fsx/describe-file-systems.html

**Comment by DrFaust92** [comment]
addressed in #12676

**Comment by bflad** [comment]
This has been released in [version 3.0.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/v3.0.0/CHANGELOG.md). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

For further feature requests or bug reports with this functionality, please create a [new GitHub issue](https://github.com/terraform-providers/terraform-provider-aws/issues/new/choose) following the template for triage. Thanks!
